### DISCUSSION TITLE
when do these events get triggered?

### DISCUSSION THREAD

**Discussion by w4hnsinn** [rakshasa/rtorrent#1140] [Q&A]
I've got a line in my .rtorrent.rc that uses `event.download.finished` as the trigger, i.e. when a torrent has finished, something happens. This is working as intended.

Now I wanted to adapt it so that the same thing happens when, for example, I "pause" a torrent, or I queue a recheck of local data, or when the recheck gets carried out.

But, simply replacing the above event with one of these did not do the trick:
```
event.download.paused
event.download.hash_done
event.download.hash_queued
```

So now I wonder if there's a specific additional condition to these that I'm not aware of, maybe.

**Comment by rakshasa** [comment]
You should at least include the relevant section of your rc file.

**Comment by w4hnsinn** [comment]
e.g.

`method.set_key = event.download.paused,"something56","f.multicall=*,f.priority.set=0;d.update_priorities="`

**Reply by rakshasa** [reply]
Use a print to verify the event gets triggered.
